### ISSUE TITLE
IPA Static Analysis Failure

### ISSUE THREAD

**Issue by ajinabraham** [MobSF/Mobile-Security-Framework-MobSF#1542]
## ENVIRONMENT

```
OS and Version: Mac OS Catalna
Python Version: Python 3.7.5
MobSF Version: 3.1.5
```

## EXPLANATION OF THE ISSUE

Non-standard IPA breaks the scan 
```
[ERROR] 18/Sep/2020 16:56:43 - Reading from Info.plist
Traceback (most recent call last):
  File "/root/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/plist_analysis.py", line 67, in plist_analysis
    dirs = os.listdir(src)
FileNotFoundError: [Errno 2] No such file or directory: '/root/.MobSF/uploads/ab66d07fa05376c8bb931e981e9e470e/Payload/'
[ERROR] 18/Sep/2020 16:56:43 - Error Perfroming Static Analysis
Traceback (most recent call last):
  File "/root/Mobile-Security-Framework-MobSF/StaticAnalyzer/views/ios/static_analyzer.py", line 102, in static_analyzer_ios
    app_dict['appstore'] = app_search(infoplist_dict.get('id'))
AttributeError: 'NoneType' object has no attribute 'get'
[ERROR] 18/Sep/2020 16:56:43 - 'NoneType' object has no attribute 'get'
[ERROR] 18/Sep/2020 16:56:43 - Internal Server Error: /StaticAnalyzer_iOS/

```

## STEPS TO REPRODUCE THE ISSUE

```
Scan the IPA available the email from Esther
```

## LOG FILE

```
Paste the contents of logs/debug.log here or attach the log file.
```

**Comment by ajinabraham** [comment]
This IPA works fine on Mac. Testing on Docker

**Comment by ajinabraham** [comment]
Looks like an issue in docker ubuntu. Will be fixed in MobSF v3.1.6
